src/SecurityManager.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Kevin;
+
+class SecurityManager
+{
+    /**
+     * Endpoint secret.
+     *
+     * @var string
+     */
+    private $endpointSecret;
+
+    /**
+     * SecurityManager constructor.
+     *
+     * @param string $endpointSecret
+     */
+    public function __construct($endpointSecret = '')
+    {
+        $this->endpointSecret = $endpointSecret;
```

Comment:
- pass endpointSecret to `verifySignature` and make it `static`
- update README.md